src/vector/platform/WebPlatform.ts

```diff
@@ -142,14 +142,9 @@ export default class WebPlatform extends VectorBasePlatform {
 
             // Set updated=1 as a query param so we can detect that we've already done this once
             // and reload the page.
-            let suffix = "updated=1";
-            if (window.location.search.length === 0) {
-                suffix = "?" + suffix;
-            } else {
-                suffix = "&" + suffix;
-            }
-
-            reloadPage(window.location.href + suffix);
+            const url = new URL(window.location.href);
+            url.searchParams.set("updated", "1");
+            window.location.href = url.toString();
```

Comment:
I think you can just do `window.location.searchParams.set("updated", "1");` here and it'll do everything in one line

Reply:
window.location isn't typed as a `URL` so that raises type errors